app/src/main/java/fr/neamar/kiss/DataHandler.java

```diff
@@ -77,7 +77,7 @@ public DataHandler(Context context) {
 
     @Override
     public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
-        if (key.startsWith("enable-")) {
+        if (key.startsWith("enable-" )) {
```

Comment:
I hate to be nitpicking, but please update your commit to not include semi-changes (such as this single added space).